src/main/java/com/couchbase/client/dcp/message/DcpOpenStreamResponse.java

```diff
@@ -30,4 +30,11 @@ public static short vbucket(ByteBuf buffer) {
         return MessageUtil.getVbucket(buffer);
     }
 
+    public static long rollbackSeqno(ByteBuf buffer) {
+        if (MessageUtil.getStatus(buffer) == 0x23) {
+            return MessageUtil.getContent(buffer).getLong(0);
+        } else {
+            throw new IllegalArgumentException("Rollback sequence number accessible only for ROLLBACK (0x23) status code");
```

Comment:
maybe do a IllegalStateException instead? since its not so much the argument is wrong but the expected state is different